src/StudioHeader.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import Responsive from 'react-responsive';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  APP_CONFIG_INITIALIZED,
+  ensureConfig,
+  mergeConfig,
+  getConfig,
+  subscribe,
+} from '@edx/frontend-platform';
+
+import DesktopHeader from './DesktopHeader';
+
+import messages from './Header.messages';
+
+ensureConfig([
+  'STUDIO_BASE_URL',
+  'LOGOUT_URL',
+  'LOGIN_URL',
+  'SITE_NAME',
+  'LOGO_URL',
+  'ORDER_HISTORY_URL',
+], 'StudioHeader component');
+
+subscribe(APP_CONFIG_INITIALIZED, () => {
+  mergeConfig({
+    AUTHN_MINIMAL_HEADER: !!process.env.AUTHN_MINIMAL_HEADER,
+  }, 'StudioHeader additional config');
+});
+
+function StudioHeader({ intl, appMenu }) {
+  const { authenticatedUser, config } = useContext(AppContext);
+
+  const mainMenu = [
+    {
+      type: 'item',
+      href: `${config.STUDIO_BASE_URL}/dashboard`,
+      content: intl.formatMessage(messages['header.links.courses']),
+    },
+  ];
```

Comment:
`mainMenu` should be an optional prop (empty by default).  In frontend-app-library-authoring, for instance, we'll need to populate it with a link to the selected library.  See here:

https://github.com/openedx/frontend-app-library-authoring/blob/master/src/library-authoring/studio-header/StudioHeader.jsx#L39-L43

Reply:
mainmenu is now an optional prop